workflows/podtato-head/workflow.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: argowf-podtato-head-chaos-
+  namespace: litmus
+spec:
+  entrypoint: argowf-chaos
+  serviceAccountName: argo-chaos
+  securityContext:
+    runAsUser: 1000
+    runAsNonRoot: true
+  arguments:
+    parameters:
+      - name: adminModeNamespace
+        value: "litmus"
+  templates:
+    - name: argowf-chaos
+      steps:
+        - - name: install-application
+            template: install-application
+        - - name: install-chaos-experiments
+            template: install-chaos-experiments
+        - - name: pod-delete
+            template: pod-delete
+        - - name: revert-chaos
+            template: revert-chaos
+          #- name: delete-application
+          #  template: delete-application
+
+    - name: install-application
+      container:
+        image: litmuschaos/litmus-app-deployer:latest
+        args: ["-namespace={{workflow.parameters.adminModeNamespace}}","-typeName=resilient","-operation=apply","-timeout=400", "-label=helloservice"]
+
+    - name: install-chaos-experiments
+      container:
+        image: alpine/k8s:1.18.2
+        command: [sh, -c]
+        args:
+          - "kubectl apply -f https://hub.litmuschaos.io/api/chaos/master?file=charts/generic/experiments.yaml -n
+            {{workflow.parameters.adminModeNamespace}} ; sleep 30"
+
+    - name: pod-delete
+      inputs:
+        artifacts:
+          - name: pod-delete
+            path: /tmp/chaosengine.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                kind: ChaosEngine
+                metadata:
+                  name: podtato-head-pod-delete-chaos
+                  namespace: {{workflow.parameters.adminModeNamespace}}
+                  labels:
+                    litmuschaos.io/engine-context: lab
+                spec:
+                  appinfo:
+                    appns: {{workflow.parameters.adminModeNamespace}}
+                    applabel: 'name=podtato-head'
+                    appkind: 'deployment'
+                  annotationCheck: 'false'
+                  engineState: 'active'
+                  chaosServiceAccount: litmus-admin
+                  monitoring: true
+                  jobCleanUpPolicy: 'retain'
+                  components:
+                    runner:
+                      imagePullPolicy: Always
+                  experiments:
+                    - name: pod-delete
+                      spec:
+                        probe:
+                        - name: "check-podtato-head-access-url"
+                          type: "httpProbe"
+                          httpProbe/inputs:
+                            url: "http://podtato-head.{{workflow.parameters.adminModeNamespace}}.svc.cluster.local:9000"
+                            insecureSkipVerify: false
+                            responseTimeout: 100
```

Comment:
Do we need to reconsider the responseTimeout value ?

Reply:
Yes @ksatchit,
Default value to be 0 and response need some time.

Reply:
by default, it will wait for few seconds. I suppose